R/constants.R

```diff
@@ -83,15 +90,31 @@ MAX_HA_TO_CROPLAND <- 3
 # Yield multiplier that goes from the observed yield to the "high" and "low" yields: observed plus or minus observed times this number
 MGMT_YIELD_ADJ <- 0.1
 
-# ======================================================================
-# energy constants
+# GDP per capita thresholds for SSP4 region groupings
+aglu.HIGH_GROWTH_PCGDP <- 12.275
+aglu.LOW_GROWTH_PCGDP  <- 2.75
+
+# Number of digits for model input data
+aglu.DIGITS_CALPRICE  <- 4 # prices and costs
+aglu.DIGITS_CALOUTPUT <- 7 # production
+
+# Regions in which agriculture and land use are not modeled
+aglu.NO_AGLU_REGIONS <- "Taiwan"
```

Comment:
It'd be great if this could be dynamic, but I suppose that is for V2 (and will only matter if we start changing regions)